internal/pkg/skuba/addons/cilium.go

```diff
@@ -1352,5 +1357,534 @@ spec:
           defaultMode: 420
           optional: true
           secretName: cilium-secret
+`
+	ciliumManifestv17 = `---
```

Comment:
Can we have the manifest on a separate file, so it is cleaner to look at the code.
It doesn't matter if the file only contains the manifest variable, it would look a lot better.

Reply:
Yes, I think it's time we have manifests per version now. Though we could do that as a tech debt cleanup, and a follow up commit.

Reply:
I've broken it out into multiple files now.